docassemble/AssemblyLine/data/questions/al_code.yml

```diff
@@ -1,11 +1,19 @@
 #############################################
 # Contains code to do various actions that
 # belong in YAML and not .py files
-# 1. Email file to court
+# 1. Email file to court TODO: does this really belong here?
 # 2. Signature flow
 ##########################################################
 # Code to handle emailing court
 ---
+# Placeholder
+objects:
+  - court_emails: DADict.using(auto_gather=False, gathered=True)
+---
+# Placeholder
+code: |
+  will_send_to_real_court = False
+---
```

Comment:
Can you comment more on this? It's not doing anything, since https://github.com/SuffolkLITLab/docassemble-AssemblyLine/blob/bd1ffb30c257887d27ee4addbafa806462190314/docassemble/AssemblyLine/al_general.py#L274 does the actual calculation

Reply:
Aha. It looks like that function is just missing from the `__all__` list. I made the wrong fix--will correct now.